src/components/Dashboard/Balances/BalancesContainer.jsx

```diff
@@ -1,13 +1,17 @@
 import React from 'react';
 import Side from './Side';
 import Balances from './Balances';
+import CommonMessage from '../../CommonMessage/CommonMessage';
```

Comment:
Not sure this level of specificity is required to import CommonMessage with the index.js within its directry.